src/main/java/iguanaman/iguanatweakstconstruct/tweaks/RepairCraftingRecipe.java

```diff
@@ -23,18 +26,33 @@ public class RepairCraftingRecipe implements IRecipe {
                 "");
     }
 
-    private ModToolRepair modifier = null;
+    private ItemModifier modifier = null;
     private ItemStack modifiedTool = null;
 
     public RepairCraftingRecipe() {
-        for (ItemModifier mod : ModifyBuilder.instance.itemModifiers) if (mod instanceof ModToolRepair) {
-            modifier = (ModToolRepair) mod;
-            break;
+        boolean gregworksLoaded = Loader.isModLoaded("TGregworks");
```

Comment:
NIT: 
1) Do this at the mod container level, once.  It might only be created once here, but this avoids people thinking this is a universal pattern.
2) Do it this way, document why it's ok in this case and not the general case so people don't copy it